radar-base.sh

```diff
@@ -168,7 +167,14 @@ time_to_update() {
 }
 
 fetch() {
-  if time_to_update; then
+  if [ -z "$1" ]; then
```

Comment:
Might want to look at default values instead of using an if. It's a way of providing a fallback, e.g.:

```
local foo = ${1:-"some default"}
```
The `:-` separates the variable name and the default to use if the variable isn't defined or is "" (empty string (since all variables in bash are really just strings (except arrays (this is getting silly now)))).